Code/GraphMol/ChemTransforms/MolFragmenter.h

```diff
@@ -118,5 +119,7 @@ RDKIT_CHEMTRANSFORMS_EXPORT std::unique_ptr<ROMol> molzip(
 RDKIT_CHEMTRANSFORMS_EXPORT std::unique_ptr<ROMol> molzip(
     const ROMol &a, const MolzipParams &params = MolzipParams());
 
+RDKIT_CHEMTRANSFORMS_EXPORT std::unique_ptr<ROMol> molzip(std::vector<ROMOL_SPTR> &decomposition,
```

Comment:
We should probably add a doc string that the decomposition is really an RGroup decomposition.

Reply:
And the Core MUST come first.

Reply:
I've added documentation here and in the Python wrappers to make this clear.